clicommand/global.go

```diff
@@ -103,6 +103,7 @@ var RedactedVars = cli.StringSliceFlag{
 		"*_PRIVATE_KEY",
 		"*_ACCESS_KEY",
 		"*_SECRET_KEY",
+		"*_CONNECTION_STRING",
```

Comment:
Makes sense. It might be worth mentioning that connection string are frequently of the form `https://username:password@host`.